src/pages/Transactions/Transaction.tsx

```diff
@@ -293,7 +296,6 @@ function renderUserTransaction(transaction: Types.UserTransaction) {
           {renderRow("Gas Used:", renderGas(transaction.gas_used))}
           {renderRow("Max Gas:", renderGas(transaction.max_gas_amount))}
           {renderRow("Gas Unit Price:", renderGas(transaction.gas_unit_price))}
-          {renderRow("Gas Currency:", transaction.gas_currency_code)}
```

Comment:
Did it get renamed or is it meant to be removed entirely?

Reply:
it has been removed https://github.com/aptos-labs/aptos-core/pull/2902